analyses/cell-type-ewings/annotate-tumor-cells.sh

```diff
@@ -30,26 +35,26 @@ sample_id="SCPCS000491" ./annotate-tumor-cells.sh
 
 set -euo pipefail
 
+# this script lives in the root of the module directory
+# use this to define other default paths
+cd $(dirname "$0")
+module_directory=$(pwd)
+
 # input variables
 sample_id=${sample_id:-"SCPCS000490"}
 normal_celltypes=${normal_celltypes:-"Endothelial cells,endothelial cell"}
 tumor_celltypes=${tumor_celltypes:-"Pulmonary vascular smooth muscle cells,smooth muscle cell"}
+data_dir=${data_dir:-"../../data/current/SCPCP000015"}
```

Comment:
Just noting for the purposes of GHA, this will still be the correct directory, as the link `current->test` will be created.

Reply:
Oh good to know!